KoalaBot.py

```diff
@@ -151,6 +153,9 @@ def load_all_cogs():
     Loads all cogs in COGS_DIR into the client
     """
     UNRELEASED = []
+    
+    #Auto generate documentation.json
+    generate_doc()

```

Comment:
generate doc would be better as its own program, run in GitHub Actions

Reply:
Code accidently added from another pr, removed